api/swagger.yml

```
@@ -2728,6 +2728,54 @@ paths:
+        200:
+          description: All roles in the system
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Role"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.38.0)

[error] too many spaces inside empty brackets

(brackets)